### ISSUE TITLE
BAT verification

### ISSUE THREAD

**Issue by LukeStaszewski** [inoda/ontrack#26]
Hey mate, sent 10 BAT to your profile on here. Only worth about $2 right now but hey ho might be worth more in the future or you could cash it out and get yourself a cheap coffee haha

Need to verify your Github with BAT Creators within 90 days to get it tho or it comes back to my wallet :)

**Comment by inoda** [comment]
@LukeStaszewski what is BAT?

**Comment by LukeStaszewski** [comment]
https://creators.brave.com/sign-up

Its a crypto that is largely focused on privacy, one of the main implementations of it is Brave Browser which is a custom Chrominum build with added privacy features and the option to view ads (privacy respecting) in turn for some of the currency. Its a cool concept, not without its downsides but its impressive as its one of the few examples of a fairly widely adopted and functional usage of blockchain and crypto and not just some fluffy idea that never came to fruition hahaha. They got some really neat anti-footprint stuff going on in the backend too.

But yeah you dont need to use the browser or anything to receive tips and can just cash em out if youd prefer normal money.

**Comment by inoda** [comment]
@LukeStaszewski I signed up but don't see any BATs. Do you know if I *have* to sign up with Uphold or Gemini as well? Or maybe it just takes a bit to go through? Anyways, thanks a lot for the contribution/thought!!! Means a lot :)

**Comment by inoda** [comment]
@LukeStaszewski after some more research, I decided not to open a Brave + Uphold account. I do have this page: https://www.buymeacoffee.com/inoda if you feel like giving back!

**Comment by LukeStaszewski** [comment]
Hi @inoda 

No worries mate - can I ask why you decided not to go with it? I know there's  some controversies around them but I've been struggling to figure out the truth, as it seems is with everything these days.

Did you have to connect an UpHold account to receive the BAT? I know that was a concern for a lot of people as it's supposed to be decentralised and yet you need to register with a central company to withdraw the funds, however I believe they are working towards fixing this amongst various regulatory issues. You can use Binance (Not sure if that's only the centralised version or their DEX decentralised exchange too) now on the main instance but it seems it's not with Brave Creators.

If you can receive the money but not  withdraw it, consider making an account under a pseudonym or something and keeping the BAT in there, hopefully there will be better ways to manage and withdraw your funds in the future if they manage to get past the regulatory issues (Providing that is the actual reason and not some corporate excuse, who knows! haha).

Anyways, I donated a coffee too via the usual route. Thanks for all your hard work and making it available to us all :D

**Comment by inoda** [comment]
@LukeStaszewski oh wow, thanks!!

I didn't go through with it because...

1. I created the Brave account and didn't see any tokens, and there was a part on my dashboard saying I needed to connect Uphold to receive payments.
2. I started making an Uphold account and they asked for my SSN, at which point I wanted to do more research. I read some posts on the BBB site about Uphold that gave me very low confidence.
